tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    "rules": {},
+    "rulesDirectory": []
+}
```

Comment:
Nit: please add a new line at the end of the file.

Reply:
Done.